File: majority-element-169/brute-force.go
// Brute-force
// Time: O(n^2)
// Space: O(1)
func majorityElement(nums []int) int {
    n := len(nums)

    if n == 1 {
        return nums[0]
    }
	
    for i:=0; i<= n/2; i++ {
        // 2
        count:= 1
        for j:=i+1; j<n; j++ {
            // main condition
            if nums[i] == nums[j] {
                count++
            }

            if count > n/2 {
                return nums[i]
            }
        }
    }
    return 0
}

File: Contains Dupicate 217/brute_force.py
# Time Complexity: O(n^2)
# Space Complexity: O(1)
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        for i in range(0, len(nums)):
            for j in range(i+1, len(nums)):
                if nums[i] == nums[j]:
                    print("True")
                    return True
        print("False")
        return False
